util/utils.go

```diff
@@ -113,7 +113,7 @@ func OS2IP(os []byte) *big.Int {
 	return new(big.Int).SetBytes(os)
 }
 
-// I2OSP converts a nonnegative integer to an octet string of a specified length.
+// IS2OP converts a nonnegative integer to an octet string of a specified length.
```

Comment:
The name should be "I2OSP" ( Integer-to-Octet-String primitive)

Reply:
please check the function name. the previous name was wrong. this is the correct one.